### PR TITLE
Adding the ability to create test entities in the DB

### PR DESCRIPTION
**Pull request by msmakouz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ class UserTableSeeder extends AbstractSeeder
 {
     public function run(): \Generator
     {
-        foreach (UserFactory::new()->times(100)->create() as $user) {
+        foreach (UserFactory::new()->times(100)->make() as $user) {
             yield $user;
         }
     }
```
Comment (butschster) [inline README.md:95]: Provide some examples of using new features

**Review by butschster — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (msmakouz) [reply README.md:129]: Readme updated

File: src/Factory/AbstractFactory.php [hunk 2]
```diff
@@ -79,14 +114,51 @@ public function createOne(): object
...
-    private function make(callable $definition): object|array
+    private function object(callable $definition): object|array
     {
         $this->entityFactory->define($this->entity(), $definition);
 
         return $this->entityFactory->of($this->entity())->times($this->amount)->make($this->replaces);
     }
 
+    /** @internal */
+    private function raw(callable $definition): array
```
[the remark this answers is not in the document]
Comment (msmakouz) [reply src/Factory/AbstractFactory.php]: Done.

File: README.md [hunk 3]
```diff
@@ -30,8 +30,8 @@ protected const LOAD = [
     \Spiral\DatabaseSeeder\Bootloader\DatabaseSeederBootloader::class,
 ];
 ```
```
[the remark this answers is not in the document]
Comment (msmakouz) [reply README.md:32]: Added info about CLI commands.

**Pull request merged**